database/migrator.go

```diff
@@ -20,7 +20,6 @@ func RunMigrations(logger log.Logger, config DatabaseConfig) error {
 	if err != nil {
 		return err
 	}
-	defer db.Close()
```

Comment:
I had to delete this line because calling `db.Close()` here will remove the InMemorySQLite db; the good news is no idle connections are left based on our tests.

Reply:
You can still do a `db.Close()` on a Sqlite3 DB as it won't remove the DB until theirs still connections to it.